convert2rhel/logger.py

```diff
@@ -91,22 +145,47 @@ def setup_logger_handler(log_name, log_dir):
     stdout_handler.setLevel(logging.DEBUG)
     logger.addHandler(stdout_handler)
 
-    # create file handler
+    # Setup a buffer for the file handler that we will add later, that way we
+    # can flush logs to the file that were logged before initializing the file handler
+    logger.addHandler(LogfileBufferHandler(100))
```

Comment:
Not something super necessary or needed, but I think adding the parameter name helps to identify what `100`means here
```suggestion
    logger.addHandler(LogfileBufferHandler(capacity=100))
```